### ISSUE TITLE
type alias docstring doesn't show in tooltip

### ISSUE THREAD

**Issue by beauxq** [microsoft/pylance-release#3402]
I found this issue https://github.com/microsoft/pylance-release/issues/1815 saying this is implemented, but it isn't working.

![type-alias-tooltip](https://user-images.githubusercontent.com/6052147/192852040-0255c9d2-df08-4f63-91f1-a3bcba5b7592.png)

## Environment data

-   Language Server version: v2022.9.40
-   OS and version: Ubuntu 20.04
-   Python version (& distribution if applicable, e.g. Anaconda): 3.8.10

## Code Snippet

```python
from typing import Dict

A = Dict[int, int]
""" xyz """
```

## Expected behavior

tooltip says "xyz"

## Actual behavior

tooltip is for `dict` class

**Comment by macintacos** [comment]
Agreed, this seems like a regression, since I know it was working before when I reported #1815. Here's a screenshot of an alias I made recently - surprised to see that it didn't show the docstring that I wrote.

<img width="731" alt="CleanShot 2022-10-08 at 23 29 33@2x" src="https://user-images.githubusercontent.com/8128941/194736454-f7e949e1-0e71-4615-8c10-eb14bc29bdad.png">

**Comment by rchiodo** [comment]
I believe this will be fixed in the next release. If you install the latest [pyright extension](https://marketplace.visualstudio.com/items?itemName=ms-pyright.pyright) this behaves as expected.

The next version of pylance will pick up any changes in pyright.

**Comment by rchiodo** [comment]
Looks like I was wrong. This is caused by pylance including Dict's doc string. This is not fixed yet.

**Comment by rchiodo** [comment]
This issue has been fixed in prerelease version 2022.11.21, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#20221121-2-November-2022-PreRelease)
